examples/typescript-interop/wasm/README.md

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
nit: remove file since it only has a todo?

Reply:
I wanted to have a file there so git actaully preserves the folder 😄